packages/core/src/common/path.ts

```diff
@@ -32,12 +32,28 @@
  * └──────┴───────────────┴──────┴─────┘
  */
 export class Path {
-    public static separator: '/' = '/';
+    static separator: '/' = '/';
 
-    public static isDrive(segment: string): boolean {
+    static isDrive(segment: string): boolean {
         return segment.endsWith(':');
     }
 
+    static normalizeDrive(path: string): string {
```

Comment:
I would like to see your [comment](https://github.com/theia-ide/theia/pull/3808#issue-237709565) (preferably with the link to the VS Code URI code), added to the PR as a JSDoc here.

Adding it as the commit comment would be great too.